docker-bake.hcl

```diff
@@ -23,10 +16,6 @@ variable "DALEC_DISABLE_DIFF_MERGE" {
     default = "0"
 }
 
-variable "DALEC_NO_CACHE_EXPORT" {
-    default = "0"
-}
-
 target "frontend" {
```

Comment:
Moving this over from the other PR, but could you add a comment here to clarify that this will build the frontend image implicitly because it will use the Dockerfile in the current dir?